### ISSUE TITLE
Change help layout to make filter usage more obvious

### ISSUE THREAD

**Issue by martinpickett** [donmelton/other_video_transcoding#120]
When running `other-transcode --help full` the "Video options" section includes several references to FFmpeg filters interspersed throughout, as well as the `--no-filters` option. To make it clearer what are filters and what are not could we have a "Video filters" section?

From what I can see the following would be in the new section: crop, 720p, 1080p, deinterlace, detelecine, no-filters & yadif-parameters.

**Comment by donmelton** [comment]
@martinpickett I will consider that. But it will take some significant code duplication and refactoring because some of those filters (i.e. `--crop` and `--720p`) are visible with the default `--help` behavior. And I don't want another section in the default.

**Comment by martinpickett** [comment]
Good point, I had not considered that. Is it easier to just move all the filters to the end of the section, then at least they are grouped, even if the group is not named?

**Comment by donmelton** [comment]
I've done that already for `--help` and `--help more`.  With `--help full` the only thing after the filters are `--rc-maxrate`, `--rc-bufsize` and `--copy-video`. And I put those last because I don't want people actually using them. :)
